pkg/api/v1/types.go

```diff
@@ -237,8 +237,8 @@ type PersistentVolumeSource struct {
 }
 
 type PersistentVolume struct {
-	TypeMeta   `json:",inline"`
-	ObjectMeta `json:"metadata,omitempty" description:"standard object metadata; see http://docs.k8s.io/api-conventions.md#metadata"`
+	runtime.TypeMeta `json:",inline"`
```

Comment:
The obvious concern here is that TypeMeta must forever and ever be backwards compatible with the oldest API version supported.  This is why all fields, even trivial constants are versioned.  Maybe this is OK here , but it makes me itchy